### ISSUE TITLE
File extension k9s seems not registered in android

### ISSUE THREAD

**Issue by guettli** [k9mail/k-9#3562]
Please search to check for an existing issue (including closed issues, for which the fix may not have yet been released) before opening a new issue: https://github.com/k9mail/k-9/issues?q=is%3Aissue

### Expected behavior

If I select a k9s file on android, I want to open it with k9.
### Actual behavior

I exported on the old android device the k9 settings to a cloud storage (nextcloud in my case)

Then I opened the file on the new android device.

Android gave me choice of some apps to open the file (with file extension k9s).
But k9 was not in the list.
That's very strange. 
I think it would be sensible if I can open a k9s file with k9 directly.

### Environment
K-9 Mail version: latest from google play store

**Comment by chrisdebian** [comment]
Hi, I'd like to work on this. Confirmed it's still a real gap — the app still exports settings with a `.k9s` filename suffix (`SettingsExporter.kt`), but there's no intent-filter registering the app to handle that file type, so tapping an exported file (e.g. after restoring from cloud storage) never offers to open it with Thunderbird.

Looking at the current settings-import screen, it only supports the user manually picking a file via `ActivityResultContracts.GetContent()` — there's no existing path for an externally-supplied file URI to flow into that same screen. So the fix looks like two parts:

1. Add an intent-filter (matching the `.k9s` extension/MIME type) so Android offers this app as a handler.
2. Wire that entry point through to the existing settings-import flow with the file URI already supplied, rather than requiring the user to also manually re-pick the file.

Before starting: does that match how you'd want this solved, or is there a preferred approach already in mind? Happy to take this on once it's assigned.
